.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 80,
```

Comment:
Does this enforce 80 character lines? If that's the case not sure if these days makes much sense. In today's displays aspect ratio vertical space is more of a premium than horizontal

Reply:
It's actually the default.

Reply:
All files are already using the 80 characters limit actually, so no real change here.